domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaimantResponse.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.cmc.domain.models.sampledata;
+
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ClaimantResponse;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ResponseAcceptation;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ResponseRejection;
+
+import java.math.BigDecimal;
+
+public abstract class SampleClaimantResponse<T extends SampleClaimantResponse<T>> {
```

Comment:
If we use the `Builder` provided by `lombok`, we will not require these sample classes with explicit builder pattern. I know we have a lot of these already but we should try to clean up, and not add more explicit builders.

Reply:
The samples provide the required fields pre-filled out to make tests easier to write

Reply:
I tried but the lombok builder doesn't work with the sample class builder method.
The builder constructs a sample object and not the actual object.

The sample project doesn't even have lombok as a dependency right now, and the sample models do use the builders on the main domain models.

The approach I have done is consistent with the full and partial admission changes which were done very recently

Reply:
ok, leave it as it is, I will clean this with others at some point. 

The point was, we don't need to introduce Builder in sample classes, instead just use the Builders from domain classes to generate sample data.